src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,76 @@
+package duke.task;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * An abstract class that represents a general task.
+ */
+public abstract class Task {
+
+    protected static final String OUTPUT_DATE_TIME_FORMAT = "yyyy/MM/dd HH:mm:ss";
+    protected static final DateTimeFormatter OUTPUT_FORMATTER = DateTimeFormatter.ofPattern(OUTPUT_DATE_TIME_FORMAT);
+    private static final String FILE_WRITING_DELIMITER = "|";
+
+    protected String taskTitle;
+    protected boolean done;
```

Comment:
Should this boolean variable sound more like a boolean?
```suggestion
    protected boolean isDone;
```